lib/index.js

```diff
@@ -41,6 +41,21 @@ Intercom.ensure(function(msg){
   }
 });
 
+Intercom.prototype.initialize = function(){
+  this.userAgent = 'Segment.io/1.0.0';
+  // Intercom asked us to split some users on the old version to help with load
+  if (this.settings.isV1Enabled) {
+    this.identify = this.identifyV1;
+    this.track = this.trackV1;
+    this.group = this.groupV1;
+  } else {
+    this.identify = this.identifyV2;
```

Comment:
A little wary about having v1/v2 as method names. Super easy to be confused here.